src/cborparser.c

```diff
@@ -915,6 +915,12 @@ static uintptr_t iterate_memcmp(char *s1, const uint8_t *s2, size_t len)
     return memcmp(s1, (const char *)s2, len) == 0;
 }
 
+static uintptr_t iterate_memcpy(char *dest, const uint8_t *src, size_t len)
```

Comment:
Yes, please return the cast return. This will still generate a function with a jmp, which is something I was trying to avoid. I wish we could detect the calling convention...